layers/+lang/elixir/packages.el

```diff
@@ -92,10 +94,13 @@
       "ta" 'alchemist-mix-test
       "tb" 'alchemist-mix-test-this-buffer
       "tt" 'alchemist-mix-test-at-point
-      "tf" 'alchemist-test-file
-      "tn" 'alchemist-test-jump-to-next-test
-      "tp" 'alchemist-test-jump-to-previous-test
+      "tT" 'alchemist-project-run-tests-for-current-file
```

Comment:
`alchemist-mix-test-file` will run tests for this file - if it cannot find the tests, it will ask to choose which test file to run.

`alchemist-project-run-tests-for-current-file` will run tests for this file - if it cannot find the tests, it will ask whether to create it (y/n) and then create the corresponding file and open it in a new window for writing.